solo/cli/key.py

```diff
@@ -310,14 +318,18 @@ def change_pin(serial):
 @click.option("-s", "--serial", help="Serial number of Solo to use")
 # @click.option("--new-pin", help="change current pin")
 def set_pin(serial):
+    client = solo.client.find(serial)
+    if client.has_pin():
+        print("This Device already has a PIN, use change-pin instead")
+        sys.exit(1)
     """Set pin of current key"""
```

Comment:
docstring should be first

Reply:
so that's a special string? I thought that was just a comment